src/utilities/alerter.ts

```diff
@@ -43,73 +56,38 @@ export function createAlerter(config: IConfig, log: Bunyan) {
 		const iAmHome = detectedMacs.has(config.myPortableDevice.mac);
 
 		if (iWasHome && !iAmHome) {
-			alertGoodbye();
+			triggerAlert('departure');

```

Comment:
Could you do something like
```suggestion
			triggerAlert(AlertType.departure);